cmake/modules/FindLibCXX.cmake

```diff
@@ -0,0 +1,15 @@
+# Find libc++
+
+find_library(LibCXX_LIBRARY NAMES c++)
+find_library(LibCXXABI_LIBRARY NAMES c++abi)
+
+find_path(LibCXX_PREFIX c++/v1/algorithm PATHS ${LibCXX_LIB_PATH}/../include)
```

Comment:
The modern CMake conventions drop `Lib` AFAIK - e.g. `find_package(ICU)` not `find_package(LibICU)`.  Please rename this to `FindCXX.cmake`.

Please don't use custom naming for the variables (`LIbCXX_LIB_PATH`).  The standard variable convention is `LibCXX_INCLUDE_DIR` (which post renaming would be `CXX_INCLUDE_DIR`) or the legacy `CXX_DIR`.  I'd change the output variable to the same name as well.